run_tests.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
+
 pytest tests/test_neutronics_utils.py -v --cov=paramak --cov-append --cov-report term --cov-report xml
```

Comment:
Just saying, you don't want the --cov-append flag at the very first line. It won't append to anything

Reply:
Ah nice spot, I will fix that

Reply:
fixed in 912d0b7